src/mjolnir/ingest_transit.cc

```diff
@@ -128,7 +128,13 @@ struct unique_transit_t {
   std::unordered_map<std::string, size_t> lines;
 };
 
-// Read from GTFS feed, sort data into the tiles they belong to
+std::string get_tile_path(const std::string tile_dir, const GraphId& tile_id) {
+  auto file_name = GraphTile::FileSuffix(tile_id);
+  file_name = file_name.substr(0, file_name.size() - 3) + "pbf";
+  return tile_dir + filesystem::path::preferred_separator + file_name;
+};
```

Comment:
pulled this out into a separate function, it's needed in multiple other functions